build.gradle

```diff
@@ -103,6 +107,30 @@ javadoc {
     }
 }
 
+
+if (rootProject.hasProperty("releaseMode")) {
+    gitPublish {
+
+        repoUri = 'https://github.com/rpmcdougall/RxJava.git'
```

Comment:
I guess this is for testing it in your own repo, right?

Reply:
Indeed, it will be changed to the proper repo when everything else is worked out.